assets/src/application/ui/layout/entityList/filterBar/style.scss

```
@@ -73,6 +79,21 @@ $filter-btn: '.esprs-button.ant-btn.ee-filter-bar__btn';
 		}
 	}
 
+	&__chain {
+		&--active {
+			&#{$filter-btn} {
+				&:focus,
+				&:hover {
+					svg {
+						path {
+							fill: var(--ee-color-white);
+						}
+					}
```

the icons still have a white background for their focus state (but change to a blue border), so using this style for `:focus` would essentially make the icon disappear

```suggestion
				&:hover {
					svg {
						path {
							fill: var(--ee-color-white);
						}
					}
```